app/controllers/users_controller.rb

```diff
@@ -15,18 +13,69 @@ def new
   def create
     @user = User.new user_params
     if @user.save
-      flash[:success] = t ".success"
+      flash[:success] = t ".account_create"
       redirect_to @user
     else
-      flash.now[:danger] = t ".fail"
+      flash.now[:danger] = t ".failed_to_create"
       render :new, status: :unprocessable_entity
     end
   end
 
+  def edit; end
+
+  def index
+    @pagy, @users = pagy User.all
+  end
+
+  def update
+    if @user.update(user_params)
+      flash[:success] = t ".profile_update"
+      redirect_to @user
+    else
+      flash[:danger] = t ".failed_to_update"
+      render "edit"
```

Comment:
```suggestion
      render :edit
```
Ưu tiên sử dụng symbol nhé